mantaray/commands.py

```diff
@@ -151,6 +151,10 @@ def msg_nickserv(view: View, core: IrcCore, message: str) -> None:
     def msg_memoserv(view: View, core: IrcCore, message: str) -> None:
         return msg(view, core, "MemoServ", message)
 
+    @add_command("/kick <nick> [<reason>]")
+    def kick(view: View, core: IrcCore, nick: str, reason: str) -> None:
+        core.kick(view.view_name, nick, reason)
```

Comment:
You should check if it is a `ChannelView`. If it isn't, `view.view_name` could be e.g. `Akuli` or `irc.libera.chat`.